src/auth/auth.action.js

```diff
@@ -5,13 +5,11 @@ import { delay, resetNavigationTo, configureLocale } from 'utils';
 import { saveLanguage } from 'locale';
 
 import {
-  fetchAccessToken,
-  fetchAuthUser,
-  fetchAuthUserOrgs,
   fetchUserOrgs,
   fetchUserEvents,
   fetchStarCount,
-} from 'api';
+} from '../user/user.api';
```

Comment:
We have [aliases](https://github.com/gitpoint/git-point/blob/master/.babelrc), so that you can replace everywhere relative paths like `../user/user.api` with `user/user.api`, should work as before.

Reply:
I learned something new, thanks @lex111 !

Reply:
Yep, all thanks to [babel-plugin-module-resolver](https://github.com/tleunen/babel-plugin-module-resolver)